bin/templates/cordova/lib/build.js

```diff
@@ -402,6 +440,16 @@ function parseOpts(options, resolvedTarget) {
                 case 'gradleArg':
                     ret.extraArgs.push(flagValue);
                     break;
+                case 'keystore':
+                case 'alias':
+                case 'keytorePassword':
+                case 'password':
+                case 'keystoreType':
+                    packageArgs[flagName] = flagValue;
+                    break;
+                case 'buildConfig':
+                    buildConfig = flagValue.replace(/"/g, '');
```

Comment:
Quotes for these reasons are generally interpreted by the shell (cmd.exe), and when we see it here, they should be gone already (and flagValue will have a space in it).